pallets/fragments/src/lib.rs

```diff
@@ -453,34 +664,36 @@ pub mod pallet {
 
 			let price: u128 = sale.price.into();
 
-			let fragment_data = <Classes<T>>::get(fragment_hash).ok_or(Error::<T>::NotFound)?;
+			let fragment_data = <Definitions<T>>::get(fragment_hash).ok_or(Error::<T>::NotFound)?;
 
-			let vault = &Self::get_vault_id(fragment_hash);
+			let vault = &Self::get_vault_id(fragment_hash); // Get the Vault Account ID of `fragment_hash`
 
 			let quantity = match options {
```

Comment:
was this fixed? The unique check? if not let's create an issue

Reply:
yeah it wasn't

Reply:
Ok let's add issues immediately from now on when we figure those things. 
I think there was another one too 🤔

Reply:
don't remember the other one 😅

Reply:
will you create it or should I? Thanks

because honestly right now I'm not even sure what the purpose of the `unique` field in `FragmentsDefinition` is apart from making sure that the `FragmentBuyOptions` parameter in `buy()` and `mint()` must be `FragmentBuyOptions::UniqueData`

😓

Reply:
was the other one the note you made on your phone, when we had our meeting?

Reply:
> will you create it or should I? Thanks
> 
> because honestly right now I'm not even sure what the purpose of the `unique` field in `FragmentsDefinition` is apart from making sure that the `FragmentBuyOptions` parameter in `buy()` and `mint()` must be `FragmentBuyOptions::UniqueData`
> 
> 😓

Actually nevermind. I guess that's all it is. 

Basically if a Fragment Definition is unique, it means that all Fragment Instances created from it require some user-inputted data. And if it's mutable, the user can change the data he inputted later on.

Nevermind, I understand it. 😅 

Do let me know if I'm wrong. Thanks 😅